public/vendor/librejs.html

```diff
@@ -46,7 +46,7 @@
           <td><a href="https://github.com/zenorocha/clipboard.js/archive/v1.5.9.tar.gz">clipboard-1.5.9.tar.gz</a></td>
         </tr>
         <tr>
-          <td><a href="./plugins/gitgraph/gitgraph.js">gitgraph.js</a></td>
+          <td><a href="../js/index.js">gitgraph.js</a></td>
```

Comment:
```suggestion
          <td><a href="../js/gitgraph.js">gitgraph.js</a></td>
```

Reply:
Thanks. I initally wanted to avoid specifying exact chunk name in case we move it later, but this makes sense.